sdks/python/apache_beam/ml/inference/pytorch_inference.py

```diff
@@ -119,7 +119,7 @@ def run_inference(
       predictions = model(batched_tensors, **prediction_params)
     return [PredictionResult(x, y) for x, y in zip(batch, predictions)]
 
-  def get_num_bytes(self, batch: List[torch.Tensor]) -> int:
```

Comment:
Do the same `Sequence` change for `sklearn_inference.py` lines 78, 89, 97, 116, 121

Reply:
Done.